src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -273,7 +273,7 @@ public void setAllowMissingReturnTag(boolean flag) {
     }
 
     /**
-     * Controls whether to ignore errors when there is no javadoc. Defaults to
+     * Controls whether to ignore violation when there is no javadoc. Defaults to
```

Comment:
plural is missing in the change. Not the first time I saw this.

Reply:
let me know if it better to be plural from English language perspective. I though that meaning is the same.

Reply:
See other review item. It is the same here. If you are referring to more than 1, it should be plural. If you are only referring to one it should have a/an before it to signify that.

Reply:
done